test/plugins/scenarios/http/config/excepted.yml

```diff
@@ -57,6 +57,21 @@ segmentItems:
               - {key: http.method, value: GET}
               - {key: url, value: 'localhost:8080/provider'}
               - {key: status_code, value: '200'}
+          - operationName: GET:/errortest
+            parentSpanId: 0
+            spanId: 2
+            spanLayer: Http
+            startTime: gt 0
+            endTime: gt 0
+            componentId: 5005
+            isError: false
```

Comment:
If we categorize status codes above 400 as errors on the server side, should we maintain the same logic on the client side?

Reply:
I'm not sure, I think we should.

Reply:
Then, please update the same logical on the client side.

Reply:
> Then, please update the same logical on the client side.

Done